docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -280,6 +294,54 @@ def configure_optimizers(self):
         scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optim, max_epochs)
         return [optim], [scheduler]
 
+class BarlowTwinsModel(BenchmarkModule):
+    def __init__(self, dataloader_kNN, num_classes):
+        super().__init__(dataloader_kNN, num_classes)
+        # create a ResNet backbone and remove the classification head
+        resnet = torchvision.models.resnet18()
+        last_conv_channels = list(resnet.children())[-1].in_features
+        self.backbone = nn.Sequential(
+            *list(resnet.children())[:-1],
+        )
+        # create a barlow twins model based on ResNet
+        self.resnet_barlowtwins = \
+            lightly.models.BarlowTwins(
+                self.backbone, 
+                num_ftrs=512,
+                proj_hidden_dim=2048,
+                out_dim=2048,
+            )
+        self.resnet_barlowtwins.projection_mlp = ProjectionHead([
```

Comment:
why don't you use the `BarlowTwinsProjectionHead`

Reply:
I did just copy-paste the setup from the ImageNette benchmark. Barlow Twins follows the SimSiam model. So for cifar10 the results seem much better by just using two instead of three layers. I wouldn't worry too much about that at this point. The goal of this PR is to get the two benchmarks more or less in sync.